float64[7] joint_angle
